cypher.y

```diff
@@ -79,15 +81,18 @@ node_properties_opt:
     ;
 
 rel_pattern:
-    rel_type rel_direction rel_type
+    rel_type
+    | rel_pattern rel_direction rel_type
     ;
 
 rel_type:
-    /* no action needed */
+    COLON IDENTIFIER
+     | LBRACKET IDENTIFIER RBRACKET
     ;
 
 rel_direction:
-    /* no action needed */
+    ARROW
+//    | ARROW rel_type_name ARROW
```

Comment:
Don't use c++ style comments.

Reply:
Removed `//` and replaced `rel_type_name` with `IDENTIFIER`